runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -867,6 +867,21 @@ else if (option.equalsIgnoreCase("use"))
 			}
 		}
 
+		if (this.swapTravel)
+		{
+			if (!(target.equals("trader crewmember") || target.equals("trader stan")) || this.charterOption.equals(CharterOption.CHARTER))
+			{
+				menuManager.addSwap("charter", target, option);
```

Comment:
Can't add swaps via menumanager like this, it will add a new swap every time a menu is generated.